AbletonMCP_Remote_Script/handlers/clips.py

```
@@ -0,0 +1,769 @@
+    """Set clip start_marker and end_marker."""
+    try:
+        clip = _get_clip(song, track_index, clip_index)
+        if start_marker is not None:
+            clip.start_marker = float(start_marker)
+        if end_marker is not None:
+            clip.end_marker = float(end_marker)
+        return {
+            "start_marker": clip.start_marker,
+            "end_marker": clip.end_marker,
+            "clip_name": clip.name,
+        }
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)